homeassistant/components/synology_dsm/translations/ca.json

```diff
@@ -30,12 +30,11 @@
                 "description": "Vols configurar {name} ({host})?",
                 "title": "Synology DSM"
             },
-            "reauth": {
+            "reauth_confirm": {
```

Comment:
Changing translation files in the backend won't have a lasting effect since we sync from Lokalise for these, not the other way around.

Reply:
my intention was, since I have only renamed the step, that I should also rename it in strings 🤔 
is it ok to leave as is?

Reply:
We should update `strings.json` but not the translation files.